src/tyr/transit_available_serializer.cc

```diff
@@ -15,20 +15,27 @@ json::MapPtr serialize(const PathLocation& location, bool istransit) {
                          {"input_lon", json::fixed_t{location.latlng_.lng(), 6}},
                          {"radius", static_cast<uint64_t>(location.radius_)}});
   json->emplace("istransit", istransit);
+
   return json;
 }
 } // namespace
 
 namespace valhalla {
 namespace tyr {
 
-std::string serializeTransitAvailable(const Api& /* request */,
+std::string serializeTransitAvailable(const Api& request,
                                       const std::vector<baldr::Location>& locations,
                                       const std::unordered_set<baldr::Location>& found) {
   auto json = json::array({});
   for (const auto& location : locations) {
     json->emplace_back(serialize(location, found.find(location) != found.cend()));
   }
+
+  // add warnings to json response
+  if (request.info().warnings_size() >= 1) {
+    json->emplace_back(json::map({{"warnings", valhalla::tyr::serializeWarnings(request)}}));
```

Comment:
we should remove `warnings` from this endpoint. you can see it returns an array of defined objects, we can't just throw a `warnings` object in the mix, that'd break the output format.